agent-install/agent-install.sh

```diff
@@ -2196,45 +2196,48 @@ function create_deployment() {
 function update_deployment() {
     log_debug "update_deployment() begin"
 
-    if $KUBECTL get deployment ${DEPLOYMENT_NAME} -n ${AGENT_NAMESPACE} >/dev/null 2>&1; then
-        # deployment exists, delete it
-        log_verbose "Found deployment ${DEPLOYMENT_NAME} in ${AGENT_NAMESPACE} namespace, deleting it..."
-        $KUBECTL delete deployment ${DEPLOYMENT_NAME} -n ${AGENT_NAMESPACE} >/dev/null 2>&1
-        chk $? 'deleting the old deployment for agent update on cluster'
-
-        # wait for the pod to terminate
-        # the 1st arg to wait_for() means keep waiting until the exit code of kubectl is exactly 1
-        if wait_for '! ( $KUBECTL -n $AGENT_NAMESPACE get pod $POD_ID >/dev/null 2>&1 || [[ $? -ge 2 ]] )' 'Horizon agent terminated' $AGENT_WAIT_MAX_SECONDS; then
-            log_verbose "Horizon agent pod terminated successfully"
-        else
-            AGENT_POD_STATUS=$($KUBECTL -n $AGENT_NAMESPACE get pod $POD_ID 2>/dev/null | grep -E '^agent-' | cut -d " " -f3)
-            if [[ $AGENT_POD_STATUS == "Terminating" ]]; then
-                if [[ $AGENT_SKIP_PROMPT == 'false' ]]; then
-                    echo "Agent pod ${POD_ID} is still in Terminating, force delete pod?[y/N]:"
-                    read RESPONSE
-                    if [[ "$RESPONSE" == 'y' ]]; then
-                        log_verbose "Force deleting agent pod ${POD_ID}..."
+    if [[ "$IS_AGENT_IMAGE_VERSION_SAME" == "true" ]] && [[ "$IS_HORIZON_ORG_ID_SAME" == "true" ]]; then
+        log_verbose "Agent image version and value of HZN_ORG_ID are same, will skip updating agent deployment"
```

Comment:
I think this is important enough information that it should be `log_info` instead of `log_verbose`. I would prefer the statement to be `log_info "Agent image version and HZN_ORG_ID are the same. Keeping the existing agent deployment."`